pages/instructorLoadFromDisk/instructorLoadFromDisk.js

```diff
@@ -53,7 +55,24 @@ var update = function(locals, callback) {
                             if (ERR(err, callback)) return;
                             if (result.hadJsonErrors) anyCourseHadJsonErrors = true;
                             debug('successfully loaded course', {courseDir});
-                            callback(null);
+                            if (config.chunksEnabled) {
+                                util.callbackify(chunks.createChunksSymlinks)({ coursePath: courseDir, courseId: result.courseId, courseData: result.courseData }, (err) => {
+                                    if (ERR(err, callback)) return;
+                                    // NOTE: Just for testing
+                                    util.callbackify(chunks.updateChunksForCourse)({
+                                        coursePath: courseDir,
+                                        courseId: result.courseId,
+                                        courseData: result.courseData,
+                                        oldHash: 'HEAD~1',
+                                        newHash: 'HEAD',
+                                    }, (err) => {
```

Comment:
Do we need this line?  I would think that we should always try to update chunks whenever we sync